md-dao/src/main/java/com/wipro/ats/bdre/md/dao/LineageNodeDAO.java

```diff
@@ -87,6 +116,18 @@ public String insert(LineageNode lineageNode) {
         return id;
     }
 
+    public LineageNode getColNodeId(String col_name, LineageNode tableNode) throws Exception {
+        //Long instanceExecId;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)